### ISSUE TITLE
It eats the last caption letter when no delimiter specified

### ISSUE THREAD

**Issue by midvok** [bicarlsen/obsidian_image_caption#10]
I just noticed an issue when no delimiter specified (default option), it seems it always eats (remove) the last caption letter.

For example, I have this code (with or without explicit image size):
```
![[IMG_20200229_124034.jpg|Home Park|400]]
# or
![[IMG_20200229_124034.jpg|Home Park]]
```
and it renders as

![](https://i.imgur.com/qTGPUTO.png)

Note, when I specified a delimiter `"` in addon's options, it works perfectly:
```
![[IMG_20200229_124034.jpg|"Home Park"|400]]
```

**Comment by bicarlsen** [comment]
Thanks so much for noticing that. It should be fixed in version `0.0.11`. Please let me know if the issue is resolved.

**Comment by midvok** [comment]
Thank you very much for the rapid fix, it works perfectly now.
